src/nvim/fileio.c

```diff
@@ -5341,10 +5352,10 @@ static int au_find_group(char_u *name)
   return AUGROUP_ERROR;
 }
 
-/*
- * Return TRUE if augroup "name" exists.
- */
-int au_has_group(char_u *name)
+/// Return true if augroup "name" exists.
+///
+/// @param name The augroup name
+bool au_has_group(char_u *name)
```

Comment:
FUNC_ATTR_PURE, possibly also FUNC_ATTR_WARN_UNUSED_RESULT (I write both, but AFAIR first implies second, at least in some compilers). Same for au_find_group, it is in the same file.